content/videos/challenges/181-image-stippling/index.json

```diff
@@ -0,0 +1,148 @@
+{
+  "title": "Weighted Voronoi Stippling",
+  "videoNumber": "181",
+  "description": "TBD",
+  "videoId": "TBD",
+  "nebulaSlug": "codingtrain-coding-challenge-181-image-stippling",
+  "date": "2024-02-22",
+  "languages": ["p5.js", "JavaScript"],
+  "topics": ["daulanay triangulation", "image stippling", "voronoi", "sweep algorithm", "Lloyd's algorithm"],
```

Comment:
Little typo, should be "Delaunay triangulation"

Reply:
Thanks!